modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc

```diff
@@ -24,8 +24,7 @@ image::update-asset-version.png[]
 +
 image::update-version-api-dc.png[]
 . Select the new version of your API specification and select *Apply and Close*.
-+
-Studio updates the API specification to the selected version and offers you to scaffold the flows again based on the new version.
-
 
+Studio updates the API specification to the selected version, and asks you if you want to regenerate the xref:4.x@apikit::apikit-4-raml-flow-concept.adoc[flow] based on the new version.
```

Comment:
We must avoid inline links as much as possible.
The concept of a flow is something that the Studio audience is already familiar with, so it's not necessary here. 

> and asks you if you want to regenerate the xref:4.x@apikit::apikit-4-raml-flow-concept.adoc[flow] based on the new version.

Should be:

> and asks you if you want to regenerate the flow based on the new version.

Reply:
Got it.

Reply:
Got it.